app/models/article.rb

```diff
@@ -7,4 +7,6 @@ class Article < ApplicationRecord
 
   validates :title, :body, presence: true
   validates :title, length: { maximum: 20 }
+
+  mount_uploader :avatar, AvatarUploader
```

Comment:
AvatarUploader is not the best name. I propose you to name it like `ImageUploader`

Reply:
Changed Avatar uploader to Image.